www/source/partials/docs/_dev-pkgs-share.html.md.erb

```diff
@@ -83,7 +85,7 @@ $ hab start core/postgresql --channel unstable
 
 An exported tarball package contains the Habitat client/binary as well as dependencies specified by your artifact.
 
-After deploying the tarball to your target server, extract the contents to the _root directory_:
+After deploying the tarball to your target server, extract the contents to the `root` directory:
```

Comment:
Well, hm. It isn't literally `root`, it's `/` (I actually stumbled on this verbiage when I read it for the first time, too.) Maybe we should just say "to the filesystem root directory (`/`)" so the user can match that up with the `/` in the example?